lightning/src/ln/chanmon_update_fail_tests.rs

```diff
@@ -1999,8 +1999,11 @@ fn test_path_paused_mpp() {
 
 	// Set it so that the first monitor update (for the path 0 -> 1 -> 3) succeeds, but the second
 	// (for the path 0 -> 2 -> 3) fails.
-	*nodes[0].chain_monitor.update_ret.lock().unwrap() = Some(Ok(()));
-	*nodes[0].chain_monitor.next_update_ret.lock().unwrap() = Some(Err(ChannelMonitorUpdateErr::TemporaryFailure));
+	chanmon_cfgs[0].persister.set_update_ret(Ok(()));
+	{
+		let mut next_update_ret = chanmon_cfgs[0].persister.next_update_ret.lock().unwrap();
+		*next_update_ret = Some(Err(ChannelMonitorUpdateErr::TemporaryFailure));
+	}
```

Comment:
Could you add a mutator in `TestPersister` for setting this? Using one prevents the `unused_mut` warning.